tests/testing/asserters_test.py

```diff
@@ -166,6 +166,37 @@ def test_assert_almost_equal(self, size, left_as_arkouda, right_as_arkouda):
                 assert_almost_equal(df, df3, atol=atol, rtol=rtol)
             assert_almost_equivalent(convert_left(df), convert_right(df3), atol=atol, rtol=rtol)
 
+    @pytest.mark.skip_if_max_rank_less_than(2)
```

Comment:
should this be `skip_if_max_rank_less_than(3)` since `shape = (2, 2, size)`?

Reply:
Good catch.  It should be fixed now.